mobify/platform-scaffold
# [Refactor] Transition from urlToPathkeys -> product IDs on SFCC

# Transition from urlToPathkeys -> product IDs on SFCC

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-112
**Linked PRs**: https://github.com/mobify/platform-scaffold/pull/711

## Changes
- Transition from urlToPathkeys -> product IDs on SFCC

## How to test-drive this PR
- Use SFCC Connectors
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Ensure things look good in the Redux store in terms of product-IDs as keys (under categories, cart, products, etc)
- Attempt a checkout process
- Navigate between PLPs and PDPs, ensuring there's no extra / duplicate keys in the store
